tests/legacy-cli/e2e/tests/web-test-runner/basic.ts

```diff
@@ -1,12 +1,29 @@
+import assert from 'node:assert';
```

Comment:
Drive by comment, one reason why a new process is used is so that changes in `process.env` do not effect other tests.